src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -74,8 +77,11 @@ The .NET Foundation licenses this file to you under the MIT license.
       <NativeFramework Include="GSS" />
     </ItemGroup>
 
+    <Exec Command="if [ -d $(NugetPackageRoot)/microsoft.dotnet.ilcompiler/7.0.0-dev/shim_source_code ]; then cd $(NugetPackageRoot)/microsoft.dotnet.ilcompiler/7.0.0-dev/shim_source_code/libs/System.Globalization.Native/  %26%26  cp CMakeLists-standalone.txt CMakeLists.txt %26%26 cp config-standalone.h config.h %26%26 mkdir -p build %26%26 cd build %26%26 cmake ../ %26%26 make -j; fi" Condition="'$(LocalSystemGlobalizationNative)' != ''"/>
```

Comment:
Would it be more maintainable if this shell code was in its own `.sh` file?

Reply:
Add separate script in next revision